packages/template-typescript-minimal/app/ssr.js

```diff
@@ -27,6 +27,9 @@ const options = {
     // except by Safari.
     protocol: 'http',
 
+    // The path that the local dev server would open initially
+    startPath: '/__pwa-kit/start',
```

Comment:
Nit: it seems like there is a small chance the Home page will result in a 404 if the value here is mismatched with the one in the path in `routes`. I don't think creating a constant would make much sense since it is likely to change when devepers starts to alternate the routes during developerment. Maybe leaving a comment in the routes to give more context for this value?

Reply:
Good point. I've added a comment in the routes file to clarify what `__pwa-kit/getting-started` path is really for.